NEWS.md

```diff
@@ -86,7 +86,7 @@
 
 ## Notes
 
-* All user-facing messages are now prepared using the `{cli}` package (#2418, @IndrajeetPatil). All messages have been reviewed and updated to be more informative and consistent.
+* All user-facing messages are now prepared using the `{cli}` package (#2418, #2643, @IndrajeetPatil). All messages have been reviewed and updated to be more informative and consistent.
```

Comment:
I would give this its own note rather than a simple issue reference -- it's a new feature, independently of {cli} being behind the implementation.